### DISCUSSION TITLE
Can not get quarkiverse server generator work

### DISCUSSION THREAD

**Discussion by fsoylemezw** [quarkusio/quarkus#43318] [Q&A]
Currently i am trying out quarkiverse generator to see if i can use it instead openapi-generator-maven-plugin.
I am following the documentation but for some reason i keep getting;

`[WARNING] [io.quarkus.config] Unrecognized configuration key "quarkus.openapi.input-base-dir" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
`

i have dependency;
`        <dependency>
            <groupId>io.quarkiverse.openapi.generator</groupId>
            <artifactId>quarkus-openapi-generator-server</artifactId>
            <version>2.4.7</version>
        </dependency>`

and plugin;

`           <plugin>
                <groupId>io.quarkus</groupId>
                <artifactId>quarkus-maven-plugin</artifactId>
                <extensions>true</extensions>
                <executions>
                    <execution>
                        <goals>
                            <goal>build</goal>
                            <goal>generate-code</goal>
                            <goal>generate-code-tests</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>`

in my pom file.What am i missing here?

**Comment by mschorsch** [comment]
@fsoylemezw I think it would be better to open a discussion/issue in https://github.com/quarkiverse/quarkus-openapi-generator.
